src/content/basics/byoc/index.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "Bring Your Own Cloud"
+description = "By default, all your tenant clusters run in the same cloud provider account. With BYOC (Bring Your Own Cloud) for AWS and Azure, you can define a specific cloud provider account to use per organization."
+date = "2018-11-16"
+weight = 50
+type = "page"
+categories = ["basics"]
++++
+
+# Bring Your Own Cloud (BYOC)
+
+The Giant Swarm architecture distinguishes between the control plane and tenant clusters, with the control plane enabling the creation and operation of tenant clusters and tenant clusters running your Kubernetes workloads.
+
+## What BYOC is good for {#benefits}
+
+Both on AWS and Azure, tenant cluster resources exist in an account (or in Azure terms: a subscription) separate from the one hosting the control plane resources. We configure a default account to use for all tenant clusters in an installation. Both accounts, the one for the control plane and the default one for tenant clusters, are in the customer's jurisdiction.
```

Comment:
Not all installations use separate accounts for control plane and tenant clusters - icconsult uses the same account for both, for example